core/whistle/include/wh_types.hrl

```diff
@@ -206,17 +206,12 @@
                              {'spawn_opt', list()}.
 -type gen_server_options() :: [gen_server_option()].
 
-%% Ibrowse-related types
--type ibrowse_error() :: {'error', 'req_timedout'
-                          | 'sel_conn_closed'
-                          | {'EXIT', any()}
-                          | {'conn_failed', {'error', atom()}}
-                         }.
--type ibrowse_ret() :: {'ok', string(), wh_proplist(), string() | binary()} |
-                       {'ibrowse_req_id', any()} |
-                       ibrowse_error().
-%% When using the stream_to option, ibrowse:send_req returns this tuple ReqID
--type ibrowse_req_id() :: {pos_integer(), pos_integer(), pos_integer()}.
+%% kz_http-related types
+-type kz_http_req_id() :: {'http_req_id', reference()} | {'ok', reference()} | reference().
+-type kz_http_ret() :: {'ok', string(), wh_proplist(), string() | binary()} |
+                       {'ok', 'saved_to_file'} |
+                       {'error', any()} |
+                       kz_http_req_id().
```

Comment:
I think we can move these to `kz_http` and `export_type`-them from there, thus using `kz_http:req_id()` instead of `kz_http_req_id()`